tests/integration/test_ses.py

```diff
@@ -181,9 +181,12 @@ def _read_message_from_filesystem(message_id: str) -> dict:
         assert "Body" not in contents2
 
         # Ensure all sent messages can be retrieved using the API endpoint
-        emails_url = config.get_edge_url() + EMAILS_ENDPOINT
```

Comment:
We need to test the 'get all messages' functionality as before.

Reply:
This is still tested in the `test_send_templated_email_can_retrospect` above.